css/style.css

```
@@ -0,0 +1,5 @@
+.container {
+    width: 1000px;
+    margin: 0 auto; 
+    text-align: center; 
+}
```

```suggestion
.container {
    width: 100%;
    max-width: 62.5rem;
    margin: 0 auto; 
    padding: 0 4rem; 
}
```
Better practice is using rem rather than **px** as **css** unit, as **px** is fixed and rem depends on root. For **container** as well as **where ever possible**, we should use **max-width** rather than fixed **width** and keep the **width** always 100%. For getting desired with we can use padding. By this the container has the fluid behavior, which by default try to adjust the available width of the device. Thanks.

---

@ich-wtag bhaiya, I resolved it and try to follow your suggestion. Thank you